libs/edit-child/feature/src/lib/edit-child.component.scss

```diff
@@ -0,0 +1,213 @@
+ion-content{
+    --ion-background-color:#EFEFEF;
+}
+
+#nav{
+    display: block;
+    width: 100%;
+    height: 60px;
+    background-color: #176560;
+}
+
+h2{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    top: 90px;
+    margin: 0;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 48px;
+    line-height: 56px;
+    text-transform: capitalize;
+    position: relative;
+    text-align: center;
+    margin-bottom: -40px;
+    color: #3C3C3C;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 48px;
+    line-height: 56px;
+    text-transform: capitalize;
+}
+
+.childrenDetails{
+    position: relative;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    top: 130px;
+}
+
+.form-details{
+    display:flex;
+    align-items: center;
+    flex-direction: column;
+    text-align: right;
+}
+
+// Button styling
+.childrenDetails input{
+    width: 411px;
+    height: 62px;
+    background: #F9F9F9;
```

Comment:
In future, this colour has now been added to themes and is called --ion-color-light-white

Reply:
It hasn't been added yet, but when it is, I will update this